linera-core/src/unit_tests/worker_tests.rs

```diff
@@ -432,7 +432,7 @@ where
     .await;
     let block_proposal = make_first_block(ChainId::root(1))
         .with_simple_transfer(ChainId::root(2), Amount::from_tokens(5))
-        .into_fast_proposal(&sender_key_pair);
+        .into_proposal_with_round(&sender_key_pair, Round::MultiLeader(0));
```

Comment:
Would it make sense to change the default in `into_fast_proposal` and `make_certificate` to be `Round::MultiLeader(0)`, or are they still used in many places where it needs to be `Fast`?

Reply:
Good point. `into_fast_proposal` is no longer used anywhere, so I could just change the default there (and the name, I guess). As for `make_certificate`, it is still being used in multiple places, but it's possible that changing the default wouldn't break those places - except a few in `test_fast_proposal_is_locked`, I guess.

Reply:
I am not sure if keeping the name (`into_fast_proposal`) while changing the implementation is a good idea - it won't create a _fast_ proposal so why call the method fast?

Reply:
I agree about the default though - let's set the round to `Multileader(0)` and maybe add an extension method that would overwrite the round if desired.

Reply:
Right, I totally missed that! Yes, it shouldn't be called "fast" anymore.